README.md

```diff
@@ -215,9 +222,6 @@ steps above:
   "ServerCertificate": "/etc/mender/server.crt"
 ```
 
-Keep in mind that `/etc/mender/mender.conf` will be overwritten if you rerun the
```

Comment:
Q: Why remove this? Is it not true anymore?

Reply:
Yep, it does not get installed anymore, at least in none of my tests.

Reply:
Yes, but split it out into a separate commit please :)